launch-dedicated.sh

```diff
@@ -23,7 +28,12 @@ ADVERTISE_ONLINE="${AdvertiseOnline:-"True"}"
 ENABLE_SINGLE_PLAYER="${EnableSingleplayer:-"False"}"
 PASSWORD="${Password:-""}"
 
-cd engine
+cd "${TEMPLATE_ROOT}"
+if [ ! -d "${ENGINE_DIRECTORY}" ]; then
+	make -f Makefile
```

Comment:
That's the idea, yeah.  If the engine doesn't exist it runs make *on the template* to compile the mod which then fetches the engine.